jsDulcificum/jsDulcificum.cpp

```diff
@@ -0,0 +1,19 @@
+#include <dulcificum.h>
+#include <emscripten/bind.h>
+#include <filesystem>
+#include <string>
+
+using namespace emscripten;
+
+extern "C"
+{
+    const std::string gcode_2_miracle_jtp(const std::string& content)
+    {
+        return dulcificum::GCode2Miracle_JTP(content);
+    }
+}
+
+EMSCRIPTEN_BINDINGS(dulcificum)
+{
+    function("gcode_2_miracle_jtp", &gcode_2_miracle_jtp);
+}
```

Comment:
I'm a bit surprised you need to redefine the `gcode_2_mircal_jpt` function here, since they have the same signature can't you directly use the function in the emscripten function?
```suggestion
EMSCRIPTEN_BINDINGS(dulcificum)
{
    function("gcode_2_miracle_jtp", &dulcificum::GCode2Miracle_JTP);
}
```
Could very well be that it is impossible to do this, but I expect you can.

Reply:
If not then it is probably a linking error and we should make things `PUBLIC` or `INTERFACE` in CMake such that the properties are transferred to the emscripten binding target

Reply:
https://github.com/Ultimaker/synsepalum-dulcificum/pull/21/commits/5ce84193f1fd10709996d225af718769683c14cf